src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents an event to be used by the Duke chat-bot.
+ */
+public class Event extends Task {
+    protected LocalDateTime from;
```

Comment:
Maybe consider using a more descriptive name for field? instead of from maybe consider startTime or fromTime